store/src/java/com/zimbra/util/ChatProvAllDomain.java

```diff
@@ -0,0 +1,88 @@
+package com.zimbra.util;
+
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2025 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.cs.account.NamedEntry;
+import com.zimbra.cs.account.Provisioning;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Utility class for retrieving and counting Zimbra domains based on the value of a specified domain feature attribute.
+ * <p>
+ * Provides methods to get all domains where a given feature is enabled and to count such domains.
+ */
+public class ChatProvAllDomain {
+
+    private static final String TRUE = "TRUE";
+
+    public static void main(String[] args) throws ServiceException {
+        Map<String, String> domains = getAllDomainsFeatureEnabled(Provisioning.A_zimbraFeatureZulipChatEnabled);
```

Comment:
Provisioning.A_zimbraFeatureZulipChatEnabled is still hard-coded, this feature name should come from args